test-framework/core/src/main/java/org/keycloak/testframework/annotations/InjectAdminClientFactory.java

```diff
@@ -15,4 +15,6 @@
     String ref() default "";
 
     LifeCycle lifecycle() default LifeCycle.CLASS;
+
+    boolean mTls() default false;
```

Comment:
I'd remove this; should just be automatic when the certificates are configured with mTLs for clients